src/components/NavBar/index.jsx

```diff
@@ -0,0 +1,21 @@
+import {
+  LOGO,
+  SEARCH,
+} from "/src/utils/constants/icons.js";
+import "./index.scss"
+
+const Navbar = () => {
+  return (
+    <div className="navbar">
+      <div className="flex navbar__container align-center justify-between">
```

Comment:
use BEM notation for naming classes throughout the project

Reply:
solvable change.